### PR TITLE
Hearing Prep Remove container unit test

### PR DESCRIPTION
**Pull request by nolawi**
#3257

### CODE REVIEW

File: client/test/karma/hearings/HearingWorksheetContainer-test.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import hearingsReducers from '../../../app/hearings/reducers/index';
+import { populateWorksheet } from '../../../app/hearings/actions/Dockets';
+import HearingWorksheetContainer from '../../../app/hearings/HearingWorksheetContainer';
+
+const store = createStore(hearingsReducers, { dockets: {} }, applyMiddleware(thunk));
+
+/* eslint-disable camelcase */
+/* eslint-disable no-unused-expressions */
+/* eslint-disable max-statements */
+describe('HearingWorksheetContainer', () => {
```
Comment (sharonwarner) [inline client/test/karma/hearings/HearingWorksheetContainer-test.js]: We should clean this up a little bit to remove these disable statements. Why does it currently need an unused expression?

File: client/test/karma/hearings/HearingWorksheetContainer-test.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        </MemoryRouter>
+      </Provider>);
+  });
+
+  afterEach(() => {
+    ApiUtilStub.afterEach();
+  });
+
+  it('retrieves a worksheet', () => {
+    setTimeout(() => {
```
Comment (nolawi) [inline client/test/karma/hearings/HearingWorksheetContainer-test.js]: `no-unused-expressions ` for the `setTimeout`

**Comment by sharonwarner** [comment]
This code generally looks good. I'm not sure if it adds any coverage that we don't already have in the feature tests though. What do you think?

**Comment by nolawi** [comment]
I agree, should we also remove https://github.com/department-of-veterans-affairs/caseflow/blob/master/client/test/karma/hearings/DailyDocketContainer-test.js to keep it consistent.

**Comment by sharonwarner** [comment]
@nolawi that sounds good. thanks!

**Comment by nolawi** [comment]
Closing this and the ticket! it looks like the autosave it tested in dockets

**Comment by nolawi** [comment]
weird, this should be passing-  closing this and reopening @sharonwarner

**Pull request closed without merging**